### PR TITLE
1530290: Remove enviroment as an input variable for the hypervisor ch…

### PR DESCRIPTION
**Pull request by wottop**
…eck in

The current server is not using this value so there is little need to send it

### CODE REVIEW

File: tests/test_config_section_libvirtd.py [hunk 1]
```diff
@@ -249,10 +227,6 @@ def test_validate_sat5_env(self):
         self.virt_config['sm_type'] = 'satellite'
         # When server is set, then owner has to be set too
         assert 'server' in self.virt_config
```
Comment (jirihnidek) [inline tests/test_config_section_libvirtd.py]: I would remove this method, because it doesn't test almost anything.

File: tests/test_fake.py [hunk 2]
```diff
@@ -118,7 +117,6 @@ def test_read_hypervisor_from_non_hypervisor(self):
 type=fake
 is_hypervisor=true
 owner=covfefe
-env=covfefe
```
Comment (jirihnidek) [inline tests/test_fake.py:121]: :-)

**Review by jirihnidek — CHANGES_REQUESTED** [review]
Thanks for PR. Overall It is good PR. I think it is necessary to remove `env` from man pages `virt-who-config.5`. BTW: there are also some obsolete CLI arguments in virt-who.zsh.

**Comment by wottop** [comment]
Will squash after second revue

**Review by jirihnidek — APPROVED** [review]
ACK. Please squash and then it is ready to be merged to master.

**Comment by wottop** [comment]
squashed

**Pull request merged**